extensions/sql-migration/src/api/azure.ts

```diff
@@ -175,3 +226,62 @@ export interface GetMigrationControllerAuthKeysResult {
 	keyName1: string,
 	keyName2: string
 }
+
+export interface GetStorageAccountAccessKeysResult {
+	keyName1: string,
+	keyName2: string
+}
+
+export interface GetMigrationControllerMonitoringData {
+	name: string,
+	nodes: MigrationControllerNode[];
+}
+
+export interface MigrationControllerNode {
+	availableMemoryInMB: number,
+	concurrentJobsLimit: number
+	concurrentJobsRunning: number,
+	cpuUtilization: number,
+	nodeName: string
+	receivedBytes: number
+	sentBytes: number
+}
+
+export interface StartDatabaseMigrationRequest {
+	location: string,
+	properties: {
+		SourceDatabaseName: string,
```

Comment:
do we need to use this casing style to work with the API?  or can we use the normal JS camelCase style.

Reply:
Yes, this is the style followed by API.

Reply:
many of the API frameworks will do auto conversion between one naming style and the other.  For example, when communicating with the C# backend the we can use JS style names in this codebase and C# style names in that codebase, and the conversion is transparent.  Have you confirmed this isn't the case with this API?